COVID_HUNGARY_REPORT.wls

```diff
@@ -262,23 +117,6 @@ sol=modelSolver[beta, initialValuesImod, alpha, alpha2, gammaa, gammas, gammah,
 initialValuesImod=Round[modelGenerator[beta, init, alpha, alpha2, gammaa, gammas, gammah, gammac, gammacr, ageDistribution, mu, ksi, asymp, pA, h][[1]]/.sol/.t->24][[1,1]]
 
 
-Evaluate[Sum[d[i][t],{i,1,nAge}]/.sol]/.t->24
-
-
-Evaluate[Sum[ic[i][t],{i,1,nAge}]/.sol]/.t->24
-
-
-(* ::Subsubsection::Closed:: *)
-(*Functions*)
-
-
-mortalityCumulative=Table[Sum[d[j][t],{j,1,i}],{i,1,nAge}];
-cumulative=Table[Sum[c[j][t],{j,1,i}],{i,1,nAge}];
-
-
-sumOfI[t_]:=Evaluate[Sum[ia1[i][t]+is1[i][t]+ia2[i][t]+is2[i][t]+ia3[i][t]+is3[i][t]+ip[i][t]+icr[i][t],{i,1,nAge}]/.sol];
-
-
 (* ::Section::Closed:: *)
 (*Scenarios post-lockdown*)
 
```

Comment:
Additionally, please keep export calls in the same section as the plot is located and right after the plot.